lld/test/ELF/systemz-pie.s

```diff
@@ -0,0 +1,59 @@
+# REQUIRES: systemz
+# RUN: llvm-mc -filetype=obj -triple=s390x-unknown-linux %s -o %t1.o
+
+## Default is no PIE.
+# RUN: ld.lld %t1.o -o %t
+# RUN: llvm-readobj --file-headers --sections -l --symbols -r %t \
+# RUN:   | FileCheck %s --check-prefix=NOPIE
+
+## Check -pie.
+# RUN: ld.lld -pie %t1.o -o %t
+# RUN: llvm-readobj --file-headers --sections -l -d --symbols -r %t | FileCheck %s
+
+## Test --pic-executable alias
```

Comment:
Delete `--pic-executable` test. Having this in `pie.s` is sufficient.

ppc32 is a good example for test organization.

Reply:
Done.